internal/controllers/phase_delete_unwanted_namespaces.go

```diff
@@ -48,12 +46,9 @@ func ensureNamespaceDeletion(ctx context.Context, c client.Client, name string)
 			Name: name,
 		},
 	}
-	err := c.Delete(ctx, namespace)
 	// don't propagate error if the Namespace is already gone
```

Comment:
nit: this comment is now in the wrong line

Reply:
It's a github glitch: 
![image](https://user-images.githubusercontent.com/20143605/123981330-c73c7900-d9b1-11eb-9ad5-2344bba1c490.png)